src/_blog/cryptotls.md

```diff
@@ -0,0 +1,60 @@
+
+To complicate matters, the quic-go API aimed to accept a regular `tls.Config` (the config struct used to configure the behavior of a TLS connection, defining, among others, the TLS certificates to use), not a type exposed by our TLS fork. This was to ensure that users of the library didn't have to create separate configs for their TCP/TLS servers and their QUIC servers. This is a pretty common use case, e.g. when running HTTP/3 and HTTP/2 in parallel. To meet this requirement, we had to use the dreaded `unsafe` package to convert between `qtls.Config` and `tls.Config`. And since there was no guarantee that the layout of these structs would remain constant between Go releases (in fact, they did change quite frequently), we had to create a new fork of crypto/tls for every new Go version (every 6 months).
+
+This meant:
+
+1. A lot of extra effort for us every time a new Go version was released. Applying the changes to the fork could get quite complicated, if there were lots of changes in crypto/tls.
+2. Every time there was a secruity-related fix in crypto/tls, we had to ask our users to update to a new (patch) release of quic-go as well.
+3. Since we couldn’t know how crypto/tls would look in future Go versions, we had to restrict the Go versions that could be used to build quic-go. This lack of forwards-compatibility prompted regular complaints from users who weren’t familiar with the internals of quic-go.
+
+This was not an ideal situation, but given the nature of QUIC, we had little choice.
+
+## Solving the Problem once and for all
+
+To tackle this situation, we joined forces with Filippo Valsorda, a former member of Google’s Go team. Although he has left the company since, he’s still maintaining crypto/tls and some other crypto packages in the standard library. Filippo, now a full-time maintainer, is sponsored by Protocol Labs for his remarkable [open-source work](https://words.filippo.io/full-time-maintainer/).
+
+Our first joint endeavor was to establish an API for crypto/tls that would enable QUIC implementations to use crypto/tls, detailed in this [GitHub issue](https://github.com/golang/go/issues/44886). This new API was designed to only support the normal (1-RTT) QUIC handshake, not the 0-RTT handshake (more on that below). 0-RTT allows clients to resume connections to servers they have previously connected to, and to send application data right in the first flight.
```

Comment:
You say "More on that below", but then explain 0-rtt. Maybe save the explanation for when you talk more about 0-rtt?